docker/Dockerfile

```diff
@@ -1,52 +1,18 @@
-################################################################################
-# Build stage 0 `builder`:
-# Install OpenSearch Benchmark from source inside a virtualenv
-################################################################################
```

Comment:
Some comments would be helpful -- "Install OpenSearch Benchmark from PyPI to build a Docker image", or something similar.